app/config.py

```diff
@@ -0,0 +1,16 @@
+from pydantic import BaseSettings
+
+
+class Settings(BaseSettings):
```

Comment:
The naming scheme here is not very indicative.
Consider more verbose names, especially if not placed in a subpackage

Reply:
this is meant to be the general configuration for all of the application.
currently only email fields are relevant.
smtp is hinting that those are email related configurations